demisto_sdk/commands/common/hook_validations/mapper.py

```diff
@@ -97,3 +101,32 @@ def is_valid_type(self):
             if self.handle_error(error_message, error_code, file_path=self.file_path):
                 return False
         return True
+
+    def is_incident_field_exist(self, id_set_file) -> bool:
+        if not id_set_file:
+            click.secho("Skipping mapper incident field validation. Could not read id_set.json.", fg="yellow")
+            return True
+
+        built_in_fields = [field.lower() for field in BUILT_IN_FIELDS] + LAYOUT_BUILT_IN_FIELDS
```

Comment:
why LAYOUT_BUILD_IN_FIELDS in mapper validator?

Reply:
done